packages/ibm-products/src/components/Tearsheet/TearsheetShell.tsx

```diff
@@ -33,6 +33,8 @@ import {
   ModalHeader,
   usePrefix,
   type ButtonProps,
+  /**@ts-ignore*/
```

Comment:
I _think_ updating our custom typings declaration file with `unstable_FeatureFlags` will remove the ts error when importing this

Reply:
https://github.com/carbon-design-system/ibm-products/blob/main/packages/ibm-products/src/custom-typings/index.d.ts